interface/forms/clinical_notes/report.php

```diff
@@ -31,38 +32,11 @@ function clinical_notes_report($pid, $encounter, $cols, $id)
         return $val['activity'] == ClinicalNotesService::ACTIVITY_ACTIVE;
     });
 
-    if ($data) {
-        ?>
-        <table class="table w-100">
-            <thead>
-            <tr>
-                <th class="border p-1"><?php echo xlt('Date'); ?></th>
-                <th class="border p-1"><?php echo xlt('Note Type'); ?></th>
-                <th class="border p-1"><?php echo xlt('Narrative'); ?></th>
-                <th class="border p-1"><?php echo xlt('Author'); ?></th>
-                <th class="border p-1"><?php echo xlt('Note Category'); ?></th>
-                <th class="border p-1"><?php echo xlt('Code'); ?></th>
-            </tr>
-            </thead>
-            <tbody>
-            <?php
-            foreach ($data as $key => $value) {
-                ?>
-                <tr>
-                    <td class="border p-1"><span class='text'><?php echo text($value['date']); ?></span></td>
-                    <td class="border p-1"><span class='text text-wrap'><?php echo text($value['codetext']); ?></span></td>
-                    <td class="border p-1"><span class='text'><?php echo text($value['description']); ?></span></td>
-                    <td class="border p-1"><span class='text'><?php echo text($value['user']); ?></span></td>
-                    <td class="border p-1"><span class='text text-wrap'><?php echo text($value['category_title']); ?></span></td>
-                    <td class="border p-1"><span class='text'><?php echo text($value['code']); ?></span></td>
-                </tr>
-                <?php
-            }
-            ?>
-            </tbody>
-        </table>
-        <?php
-    }
-}
+    $viewArgs = [
+        'notes' => $data
```

Comment:
In the previous code there was an `if` guarding against falsy values of `$data`. What happens if a falsy value is passed here? Could there be any errors?

Reply:
Shouldn't be a problem, however, I can always define the data variable prior to the query and only append if there are results.

Reply:
Actually, it looks like data is already returning an array regardless of whether there are notes.